rules/docker-fpm-frr.mk

```diff
@@ -31,3 +31,4 @@ $(DOCKER_FPM_FRR)_BASE_IMAGE_FILES += TSA:/usr/bin/TSA
 $(DOCKER_FPM_FRR)_BASE_IMAGE_FILES += TSB:/usr/bin/TSB
 $(DOCKER_FPM_FRR)_BASE_IMAGE_FILES += TSC:/usr/bin/TSC
 $(DOCKER_FPM_FRR)_BASE_IMAGE_FILES += monit_bgp:/etc/monit/conf.d
+$(DOCKER_FPM_FRR)_BASE_IMAGE_FILES += $(SUPERVISOR_PROC_EXIT_LISTENER_SCRIPT)
```

Comment:
Check builds are failing because `SUPERVISOR_PROC_EXIT_LISTENER_SCRIPT` should be added to `$(DOCKER_FPM_FRR)_FILES`, not `$(DOCKER_FPM_FRR)_BASE_IMAGE_FILES`

Reply:
Fixed.